daemon/Makefile

```diff
@@ -9,6 +9,7 @@ GO_BINDATA := go-bindata -prefix ../ -mode 0640 -modtime 1450269211 \
 	-ignore netdev_config.h -ignore node_config.h -ignore '.+\.o$$' \
 	-ignore '.+\.orig$$' -ignore '.+~$$' -ignore '\.DS_Store'
 GO_BINDATA_SHA1SUM=4c07b9aaa78073a929aa67dc4bcb862de58d7522
+GO_VERSION_USED=go1.8.3
```

Comment:
Is the expectation that we notice incompatible go versions by seeing this value changing in a patch?